drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -1024,32 +1010,8 @@ function DragAndDropBlock(runtime, element, configuration) {
      * We have to do this in JS, not python, since some migrations depend on the image size,
      * which is not known in Python-land.
      */
-    var migrateState = function(bg_image_width, bg_image_height) {
-        Object.keys(state.items).forEach(function(item_id) {
-            var item = state.items[item_id];
-            if (item.x_percent === undefined) {
-                // Find the matching item in the configuration
-                var width = 190;
-                var height = 44;
-                for (var i in configuration.items) {
-                    if (configuration.items[i].id === +item_id) {
-                        var size = configuration.items[i].size;
-                        // size is an object like '{width: "50px", height: "auto"}'
-                        if (parseInt(size.width ) > 0) {  width = parseInt(size.width); }
-                        if (parseInt(size.height) > 0) { height = parseInt(size.height); }
-                        break;
-                    }
-                }
-                // Update the user's item state to use centered relative coordinates
-                var left_px = parseFloat(item.left) - 220; // 220 px for the items container that used to be on the left
-                var top_px = parseFloat(item.top);
-                item.x_percent = (left_px + width/2) / bg_image_width * 100;
-                item.y_percent = (top_px + height/2) / bg_image_height * 100;
-                delete item.left;
-                delete item.top;
-                delete item.absolute;
-            }
-        });
+    var migrateState = function() {
```

Comment:
@e-kolpakov OK, that's a bit unusual but I get where you're coming from. If you're not going to remove the method, could you update the docstring above to mention why it's there (and remove the part about image size)?